hoomd/md/EvaluatorPairALJ.h

```diff
@@ -1635,7 +1661,7 @@ template<> std::string EvaluatorPairALJ<3>::getShapeSpec() const
     std::ostringstream shapedef;
     const ManagedArray<vec3<Scalar>>& verts(shape_i->verts);
     const unsigned int N = verts.size();
-    if (N == 0)
+    if (N < 2)
```

Comment:
```suggestion
    if (N == 2)
```

Reply:
Should this suggestion say, ``if N==1``?

Reply:
Yes it should.